File: app/src/main/java/com/example/bervianto/ecceg_rsa_app/MainActivity.java
package com.example.bervianto.ecceg_rsa_app;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import butterknife.ButterKnife;
import butterknife.OnClick;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ButterKnife.bind(this);
    }

    @OnClick(R.id.rsa_button)
    void launchRSA() {
        Intent intent = new Intent(this, RSAActivity.class);
        startActivity(intent);
    }

    @OnClick(R.id.ecceg_button)
    void launchECCEG() {
        Intent intent = new Intent(this, ECCEGActivity.class);
        startActivity(intent);
    }
}

File: app/src/main/java/id/my/berviantoleo/ecceg_rsa_app/MainActivity.kt
package id.my.berviantoleo.ecceg_rsa_app

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import id.my.berviantoleo.ecceg_rsa_app.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.rsaButton.setOnClickListener {
            launchRSA()
        }

        binding.eccegButton.setOnClickListener {
            launchECCEG()
        }
    }

    private fun launchRSA() {
        val intent = Intent(this, RSAActivity::class.java)
        startActivity(intent)
    }

    private fun launchECCEG() {
        val intent = Intent(this, ECCEGActivity::class.java)
        startActivity(intent)
    }
}
